specs/language/introduction.tex

```diff
@@ -54,6 +54,47 @@
 where this document diverges this section, the remaining sections in this
 chapter, and the attached \ref{glossaries}.
 
+\Sec{Common Definitions}{Intro.Defs}
+
+\p The following definitions are consistent between \acrshort{hlsl} and the
+\gls{isoC} and \gls{isoCPP} specifications, however they are included here for
+reader convenience.
+
+\Sub{Diagnostic Message}{Intro.Defs.Diags}
+\p An implementation defined message belonging to a subset of the
+implementation's output messages which communicates diagnostic information to
+the user.
+
+\Sub{Ill-formed Program}{Intro.Defs.IllFormed}
+\p A program that is not well formed, for which the implementation is expected
+to return unsuccessfully and produce one or more diagnostic messages.
+
+\Sub{Implementation-defined Behavior}{Intro.Defs.ImpDef}
+\p Behavior of a well formed program and correct data which may vary by the
```

Comment:
Nit: I think "well-formed" is a more accepted spelling, similar to your use of "ill-formed" elsewhere.